smartBugs.py

```diff
@@ -16,22 +16,22 @@
 from time import time, localtime, strftime
 
 
-cfg_dataset_path = os.path.abspath('config/dataset/dataset.yaml')
+cfg_dataset_path = os.path.abspath(os.path.dirname(os.path.realpath(__file__)) + '/config/dataset/dataset.yaml')
 with open(cfg_dataset_path, 'r') as ymlfile:
     try:
         cfg_dataset = yaml.safe_load(ymlfile)
     except yaml.YAMLError as exc:
         print(exc)
```

Comment:
do you wanna just print, or also exit?

Reply:
I would actually suggest to use some logging mechanism, instead of prints.

https://docs.python.org/3/howto/logging.html